src/Interfaces/IGlacisAirlift.sol

```diff
@@ -0,0 +1,87 @@
+// SPDX-License-Identifier: MIT
+/// @custom:version 1.0.0
+pragma solidity ^0.8.17;
+
+struct QuoteSendInfo {
```

Comment:
if there are parameter descriptions available (e.g. in the original interface contract of the third party) then it would be nice to copy them over here, too.

Reply:
also looked at this but there are no descriptions for these structs :/